packages/graphql/src/schema-model/parser/parse-arguments.ts

```diff
@@ -18,7 +18,7 @@
  */
 
 import { inspect } from "@graphql-tools/utils";
-import type { Maybe } from "@graphql-tools/utils/typings/types";
+import type { Maybe } from "@graphql-tools/utils/";
```

Comment:
Weird trailing slash?

```suggestion
import type { Maybe } from "@graphql-tools/utils";
```